R/zzx-deprecated.R

```diff
@@ -403,7 +403,7 @@ cdm_learn_from_db <- function(dest) {
 #' @export
 cdm_check_constraints <- function(dm) {
   deprecate_soft("0.1.0", "dm::cdm_check_constraints()", "dm::dm_examine_constraints()")
-  dm_examine_constraints_impl(dm = dm)
+  dm_examine_constraints_impl(dm = dm, progress = FALSE)
```

Comment:
Do we still need this?

Reply:
Is it a question for me ? progress is NA by default, so by setting to FALSE I make sure old behavior is unchanged. Or do you mean to hard deprecate the function ?

Reply:
I'd expect that tests run in noninteractive mode, so this would not be necessary. But we can leave it as is.